shared/src/main/scala/fm/common/rich/RichOption.scala

```diff
@@ -36,4 +37,9 @@ final class RichOption[A](val self: Option[A]) extends AnyVal {
       case None    => Future.successful(None)
     }
   }
+
+  /**
+   * Implements asJava method similar to collection.JavaConverters._ for scala Option class
+   */
+  def asJava: Optional[A] = Optional.ofNullable(self.getOrElse(null).asInstanceOf[A])
```

Comment:
This produces a warning:
```
[warn] /Users/tim/fm/fm-common/shared/src/main/scala/fm/common/rich/RichOption.scala:44:82: a type was inferred to be `Any`; this may indicate a programming error.
[warn]   def asJava: Optional[A] = Optional.ofNullable(self.getOrElse(null).asInstanceOf[A])
```